src/metabase/driver/sql_jdbc/execute.clj

```diff
@@ -714,6 +712,29 @@
               results-metadata {:cols (column-metadata driver rsmeta)}]
           (respond results-metadata (reducible-rows driver rs rsmeta qp.pipeline/*canceled-chan*))))))))
 
+(defn sql->reducible-rows
+  "Returns a reducible collection of rows from `db` and a given SQL query."
+  [db [sql & params]]
+  (let [driver (:engine db)]
+    (reify clojure.lang.IReduceInit
+      (reduce [_ rf init]
+        (do-with-connection-with-options
+         driver
+         db
+         nil
+         (fn [^Connection conn]
+           (with-open [stmt          (statement-or-prepared-statement driver conn sql params nil)
+                       ^ResultSet rs (try
+                                       (let [max-rows 0] ; 0 means no limit
```

Comment:
> ; 0 means no limit

is this from jdbc or it's driver-specific?

Reply:
JDBC